src/helpers/consts.js

```diff
@@ -258,6 +260,15 @@ export const COOKIE_UI_LANG    = 'archive_UIlang';
 export const LANG_UI_LANGUAGES = [LANG_HEBREW, LANG_ENGLISH, LANG_RUSSIAN, LANG_SPANISH, LANG_GERMAN, LANG_UKRAINIAN];
 
 // Search
+export const ES_RESULT_TYPE_SOURCES = "sources"
+export const ES_RESULT_TYPE_TAGS = "tags"
+
+// Blog
+export const BLOG_ID_LAITMAN_RU = 1
```

Comment:
Please make the code in the `blog posts` and `tweets` tabs use these constants as well so the code base is uniform